x-pack/plugins/ml/common/types/alerts.ts

```diff
@@ -108,3 +108,36 @@ export type MlAnomalyDetectionAlertRule = Omit<Alert<MlAnomalyDetectionAlertPara
 export interface JobAlertingRuleStats {
   alerting_rules?: MlAnomalyDetectionAlertRule[];
 }
+
+interface CommonHealthCheckConfig {
+  enabled: boolean;
+}
+
+export type MlAnomalyDetectionJobsHealthRuleParams = {
+  includeJobs: {
+    jobIds?: string[];
+    groupIds?: string[];
+  };
+  excludeJobs?: {
+    jobIds?: string[];
+    groupIds?: string[];
+  };
+  testsConfig?: {
+    dataFeed?: CommonHealthCheckConfig | null;
```

Comment:
`dataFeed` shouldn't be camelcase.

```suggestion
    datafeed?: CommonHealthCheckConfig | null;
```

Reply:
Renamed in https://github.com/elastic/kibana/pull/106084/commits/b08031efbc265847be7d3617ca90253110b74fc9